wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php

```diff
@@ -106,8 +119,9 @@ public function finishPanel()
         } ?>
         <div class="wrap">
             <div class="finish-setup-content">
+                <span id="finish-setup-dismiss" class="notice-dismiss"><span class="screen-reader-text"><?php _e("Dismiss", 'cds'); ?></span></span>
```

Comment:
The button is kind of low contrast-y, but we are admins so no big deal really.

Reply:
Ya just using whatever is built in.

Reply:
Fixed.